pkg/project/create.go

```diff
@@ -79,7 +80,11 @@ func Create(templateName string, home string) error {
 				return err
 			}
 			slog.Info("patching", "file", patchStep.File)
-			data, err := os.ReadFile(patchStep.File)
+			b, err := os.ReadFile(patchStep.File)
+			if err != nil {
+				return err
+			}
+			data, err := hujson.Standardize(b)
```

Comment:
can you send this in another PR

Reply:
@thdxr https://github.com/sst/ion/pull/88